frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts

```diff
@@ -0,0 +1,108 @@
+  obcCount,
+  resourceProviders,
+  savingsValue,
+} from '../views/noobaaDashboardPage.view';
+
+const testBucket = {
+  apiVersion: 'objectbucket.io/v1alpha1',
+  kind: 'ObjectBucketClaim',
+  metadata: {
+    namespace: testName,
+    name: 'test-bucket',
+  },
+  spec: {
+    ssl: false,
+    storageClassName: 'openshift-storage.noobaa.io',
+    generateBucketName: 'test-bucket',
+  },
+};
+
+const loadAndWait = async () => {
+  await browser.get(`${appHost}/dashboards/object-service`);
+  await browser.wait(until.and(untilNoLoadersPresent));
+};
+
+describe('Test Buckets Card', () => {
+  beforeEach(async () => {
+    await loadAndWait();
+  });
+
+  it('Test at least one noobaa bucket is present', async () => {
+    await browser.wait(until.visibilityOf(noobaaCount));
+    const text = await noobaaCount.getText();
+    const [count, ,] = text.split(' ');
+    expect(Number(count)).toBeGreaterThanOrEqual(1);
+  });
+
+  it('Create an Object Bucket Claim and test equality', async () => {
+    await browser.wait(until.visibilityOf(obcCount));
+    let text = await obcCount.getText();
+    const initialCount = text.substring(0, 1);
+    execSync(`echo '${JSON.stringify(testBucket)}' | kubectl create -f -`);
+    await browser.wait(until.and(waitFor(obcCount, (initialCount + 1).toString())));
+    text = await obcCount.getText();
+    const finalCount = text.substring(0, 1);
+    expect(Number(finalCount)).toEqual(Number(initialCount) + 1);
+    execSync(`echo '${JSON.stringify(testBucket)}' | kubectl delete -f -`);
+  });
+});
+
+describe('Test Status Card', () => {
+  beforeEach(async () => {
+    await loadAndWait();
+  });
+
+  it('Check if Multi Cloud Gateway is in a healthy state', async () => {
+    await browser.wait(until.visibilityOf(healthOfMCG));
+    expect(healthOfMCG.isPresent()).toBe(true);
+  });
+});
+
+describe('Test Object Data Reduction Card', () => {
+  beforeEach(async () => {
+    await loadAndWait();
+  });
+
+  it('Check if Efficiency Ratio is in acceptable data range', async () => {
+    await browser.wait(until.visibilityOf(efficiencyValue));
+    const effValue = await efficiencyValue.getText();
+    const [ratioA, ratioB] = effValue.split(':');
+    const [numA, numB] = [Number(ratioA), Number(ratioB)];
+    expect(numA).toBeGreaterThan(0);
+    expect(numB).toBeGreaterThan(0);
+  });
+
+  it('Check for savings value to be in acceptable data range', async () => {
+    await browser.wait(until.visibilityOf(savingsValue));
+    const savVal = await savingsValue.getText();
+    const [savDigits, ,] = savVal.split('%');
+    const numSav = Number(savDigits);
+    if (Number.isNaN(numSav)) {
+      expect(savVal.trim()).toEqual('No Savings');
+    } else {
+      expect(numSav).toBeGreaterThanOrEqual(0);
+    }
+  });
+});
+
+describe('Test Resource Providers card', () => {
+  beforeEach(async () => {
+    await loadAndWait();
+  });
+
+  it('Check if resource provider has at least 1 AWS resource', async () => {
```

Comment:
As said it can be on other infra, you should check if it has atleast one resource, not for any specific resource.
```suggestion
  it('Check if resource provider has at least 1 resource', async () => {
```

Reply:
In VMWare would there be a resource by default? If so then which cloud provider?

Reply:
Tests should not care about that, but rather on the count. There will always be one default backing store wherever the noobaa is deployed with ocs. On vmware it will be ceph.

Reply:
Are you creating these test cases by yourself ?